mocks/log.go

```diff
@@ -126,47 +126,47 @@ func (msw *MockSyslogWriter) Clear() {
 }
 
 // Close releases resources. No other methods may be called after this.
-func (msw *MockSyslogWriter) Close() error {
```

Comment:
I'm a little hesitant about these MockSyslogWriter pointer changes. I think this was clean up? Maybe it fixed a bug I can't see! 

If it's the former, I'm less chill about it going in here (esp with the NewMockSyslog returning a pointer type and with the log on the audit struct needing to be cast by hand currently).